packages/ui-extensions/src/surfaces/admin/components/DateField/examples/range-datefield.example.ts

```diff
@@ -3,7 +3,10 @@ import {extend, DateField} from '@shopify/ui-extensions/admin';
 extend('Playground', (root) => {
   const dateField = root.createComponent(
     DateField,
-    { selected: {start: '2023-11-08', end: '2023-11-10' } },
+    {
+      label: 'DateField',
+      value: {start: '2023-11-08', end: '2023-11-10'},
```

Comment:
This isn't valid for DateField - it expects only a string. We should delete this example.